benchmarks/benchmarks/utils.py

```diff
@@ -534,7 +534,7 @@ def skip_if_not_4gpu():
     """skip if DGL_BENCH_DEVICE is gpu"""
 
     def _wrapper(func):
-        if GPU_COUNT != 4:
+        if GPU_COUNT < 4:
```

Comment:
Why this change?

Reply:
because I think any instance with no less than 4 GPUs can run these tests, no need to be exactly equal `4`.